app/views/lead_providers/report_schools/choose_delivery_partner.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :title, "Choose delivery partner" %>
+<% content_for :before_content, govuk_back_link( text: "Back", href: start_lead_providers_report_schools_path) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Choose the delivery partner</h1>
+    <p>Select the delivery partner that these schools have confirmed an agreement with.</p>
+    <%= form_with model: @delivery_partner_form, url: check_delivery_partner_lead_providers_report_schools_path, method: :post do |f| %>
+      <%= f.govuk_collection_radio_buttons(
+        :delivery_partner_id,
+        @delivery_partners,
+        :id,
+        :name,
+        legend: nil) %>
```

Comment:
Hmm, I wonder if we should make the paragraph above into the `legend`.

Reply:
I could but these legends use a header tag by default and I find what we've been doung to use a p tag quite fiddly and difficult to understand when reading the code without offering any clear benefit.